### ISSUE TITLE
The pros & cons section of design consideration appears to be redundant?

### ISSUE THREAD

**Issue by E0735389** [nus-cs2103-AY2324S2/forum#629]
The example undo/redo feature has this.

![image](https://github.com/nus-cs2103-AY2324S2/forum/assets/97508964/455f18fd-d59f-4e23-a89b-affdeddd31a8)

However, isn't the description quite redundant? Because the pros of alternative 1 is exactly is cons of alternative 2, and vice versa.

Wouldn't it be simpler to only list the pros of each alternative, and the cons is automatically inferred as the pros of the other alternatives?

**Comment by damithc** [comment]
@E0735389 You are right, this part can be shortened by eliminating the redundancy you have pointed out.
